libraries/plugins/peerplays_sidechain/sidechain_net_handler.cpp

```diff
@@ -68,7 +68,7 @@ void sidechain_net_handler::sidechain_event_data_received(const sidechain_event_
    const chain::global_property_object &gpo = database.get_global_properties();
 
    // Deposit request
-   if ((sed.peerplays_to == gpo.parameters.son_account()) && (sed.sidechain_currency.compare("1.3.0") != 0)) {
+   if ((sed.peerplays_from == gpo.parameters.son_account()) && (sed.sidechain_currency == "BTC")) {
 
```

Comment:
Condition is not good, it should be compared to the asset ID string, not asset SYMBOL. This breaks depositing peerplays assets.